src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/SymbolKey/SymbolKey.cs

```diff
@@ -211,9 +219,19 @@ internal static string CreateStringWorker(int version, ISymbol? symbol, Cancella
         /// <paramref name="compilation"/> to a matching symbol.
         /// </summary>
         public SymbolKeyResolution Resolve(
-            Compilation compilation, bool ignoreAssemblyKey = false, CancellationToken cancellationToken = default)
+            Compilation compilation, bool ignoreAssemblyKey, CancellationToken cancellationToken)
+        {
+            return Resolve(compilation, ignoreAssemblyKey, ignoreReturnTypes: false, cancellationToken);
+        }
+
+        /// <summary>
+        /// Tries to resolve this <see cref="SymbolKey"/> in the given 
+        /// <paramref name="compilation"/> to a matching symbol.
+        /// </summary>
+        public SymbolKeyResolution Resolve(
+            Compilation compilation, bool ignoreAssemblyKey = false, bool ignoreReturnTypes = false, CancellationToken cancellationToken = default)
```

Comment:
if we can get rid of ignoreReturnTypes , then i would really like to

Reply:
Yeah, I just flipped this quickly to see if the build would pass... and it didn't. Will investigate on Monday